src/index.js

```diff
@@ -29,9 +29,11 @@ export default async function track({ services = [], termsTypes: documentTypes,
     });
   }
 
-  await archivist.refilterAndRecord(serviceIds, documentTypes);
+  // The result of the extraction step that generates the version from the snapshots may depend on changes to the engine or its dependencies.
+  // So, let's start by only performing the extraction process so that we can annotate any versions generated related to such changes and avoid sending notifications.
```

Comment:
Avoid the use of “we”, as we have decided for the documentation.

```suggestion
  // The process thus starts by only performing the extraction process so that any version following such changes can be labelled (to avoid sending notifications, for example).
```